LICENSE

```diff
@@ -295,151 +295,147 @@ Dual licensed under the MIT and GPL licenses.
-         
-        2. Redistributions in binary form must reproduce the above copyright notice,
-        this list of conditions and the following disclaimer in the documentation and/or
-        other materials provided with the distribution.
-         
-        THIS SOFTWARE IS PROVIDED BY OPENLAYERS CONTRIBUTORS ``AS IS'' AND ANY EXPRESS
-        OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
-        MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
-        SHALL COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
-        INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
-        LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
-        PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
-        LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
-        OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
-        ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-         
-        The views and conclusions contained in the software and documentation are those
-        of the authors and should not be interpreted as representing official policies,
-        either expressed or implied, of OpenLayers Contributors.
-
-  Includes compressed code under the following licenses:
-
-  (For uncompressed versions of the code used please see the
-  OpenLayers SVN repository: <http://openlayers.org/>)
+* OpenLayers.js -- OpenLayers Map Viewer Library *
 
-    -------------------------------------------------------
-      Contains portions of Rico <http://openrico.org/>
-
-      Copyright 2005 Sabre Airline Solutions
-
-      Licensed under the Apache License, Version 2.0 (the "License"); you
-      may not use this file except in compliance with the License. You
-      may obtain a copy of the License at the top of this file or a the following
-      link:
-  
-             http://www.apache.org/licenses/LICENSE-2.0
-  
-      Unless required by applicable law or agreed to in writing, software
-      distributed under the License is distributed on an "AS IS" BASIS,
-      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
-      implied. See the License for the specific language governing
-      permissions and limitations under the License.
+Copyright (c) 2006-2012 by OpenLayers Contributors
+Published under the 2-clause BSD license.
 
-    -------------------------------------------------------
-      Contains XMLHttpRequest.js
-      <https://code.google.com/p/xmlhttprequest/source/browse/trunk/source/XMLHttpRequest.js?r=29>
-      Copyright 2007 Sergey Ilinsky (http://www.ilinsky.com)
+Full license text below taken from https://github.com/openlayers/openlayers/blob/release-2.12/license.txt
+(Tag release-2.12 corresponded to commit 0412410be0ec43dc9a5c258b640a793fad41bd88 on May 16, 2015).
```

Comment:
Oh, I get it - that was the date you verified the tag - I misread and interpreted it as the date of the commit :smile:.